src/app/core/core.module.ts

```diff
@@ -1,9 +1,16 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { VehiclesRestService } from './vehicles/vehicles-rest.service';
+import { VehiclesService } from './vehicles/vehicles.service';
+import { HttpClientModule } from '@angular/common/http';
 
 @NgModule({
   imports: [
-    CommonModule
+    CommonModule,
+    HttpClientModule
+  ], 
+  providers: [
+    { provide: VehiclesService, useClass: VehiclesRestService },
```

Comment:
Jeden provider w app.module, teraz drugi tutaj. What's going on?

Reply:
Usunięto ten w `app.module`, pozostał w `core`.